packages/prepo-stores/src/utils/CustomProviders.ts

```diff
@@ -18,23 +18,33 @@ export class TenderlyRpcProvider extends ethers.providers.JsonRpcProvider {
 }
 
 type FallthroughProviderOptions = {
+  includeWalletProvider?: boolean
```

Comment:
I think simpler if this is `excludeWalletProvider`, then in constructor we don't need to re-assign to option parameter and can remove the DEFAULT_OPTIONS because excludingWalletProvider is falsy by default

```suggestion
  excludeWalletProvider?: boolean
```